owner/src/main/java/org/the_chance/honeymart/ui/add_product/components/ImageBox.kt

```diff
@@ -0,0 +1,58 @@
+package org.the_chance.honeymart.ui.add_product.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Clear
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import coil.compose.rememberAsyncImagePainter
+import org.the_chance.honymart.ui.composables.IconButton
+import org.the_chance.honymart.ui.theme.black16
+import org.the_chance.honymart.ui.theme.dimens
+import org.the_chance.design_system.R
+
+@Composable
+fun ImageBox(
```

Comment:
I see this name no clean this composable use as item need rename **itemImageProdcte** same this style